k9mail/src/main/java/com/fsck/k9/activity/setup/FolderSettings.java

```diff
@@ -27,7 +27,7 @@
 
     private static final String PREFERENCE_TOP_CATERGORY = "folder_settings";
     private static final String PREFERENCE_DISPLAY_CLASS = "folder_settings_folder_display_mode";
-    private static final String PREFERENCE_SYNC_CLASS = "folder_settings_folder_sync_mode";
+    private static final String PREFERENCE_POLL_CLASS = "folder_settings_folder_poll_mode";
```

Comment:
In general, changing a preference key without migration code will change the behavior of existing installations, i.e. the existing setting value will be lost. Folder settings in K-9 Mail are weird because we save them in two places. So this might not actually break with existing installations. But it's still not the right thing to do.